src/components/setLocalStorage.js

```diff
@@ -0,0 +1,8 @@
+export const setLocalStorage = (questionNum) => {
+  document.querySelectorAll('li').forEach((answer) => {
```

Comment:
This file is not a component, so I would say you have two options to
- move this function to another file where it is used
- create a folder called `utils` and put this file there

`utils` means utilities usually represent all files that has helper functions

Reply:
Sure we can use second option I will do that